### PR TITLE
Module to convert file directory to JSON

### PR DESCRIPTION
**Pull request by Timothylock**
This looks familiar :) 

The ID Significant changes can use the same module as the Locate IR points with the difference that all the images should be fed into it. This module is a "hack" and given a directory, will generate an output json that can be fed into the Locate IR points module.

### CODE REVIEW

**Comment by Timothylock** [comment]
README will come before merge! And I mean it this time lol

File: src/task1/sig-changes-json-file-listing/listdirjson.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#
+#{
+#  "positive": ["../common/test/2018-05-25_16-22-29-018.bmp", "../common/test/2018-05-25_16-22-29-685.bmp"]
+#}
+#
+# for the ID Significant Changes module to use since its just IR Location module repurposed for identifying significant changes
+
+# Set up the command line argument parsing
+def define_args():
+    parser = argparse.ArgumentParser(description='Detect broken solar panels')
```
Comment (xiejxie) [inline src/task1/sig-changes-json-file-listing/listdirjson.py:15]: ;)

**Review by xiejxie — CHANGES_REQUESTED** [review]
Btw should this be an entire module on it's own? Wouldn't it be better to have it as a function in some common file?

**Comment by Timothylock** [comment]
@xiejxie  Is this still needed? If not, feel free to close this PR

**Comment by Timothylock** [comment]
Closing

**Pull request closed without merging**